### PR TITLE
Feature gallery

### PR DESCRIPTION
**Pull request by DevStarlight**

### CODE REVIEW

**Comment by DevStarlight** [comment]
After trying lots of things, I found that horizontal [infinite scrolling](https://github.com/sroze/ngInfiniteScroll) is not supported by this library yet. It seems someone did [that](https://github.com/sroze/ngInfiniteScroll/pull/142) feature but the branch is not merged to the master branch since it has no test

File: sass/base/_variables.scss [hunk 1]
```diff
@@ -16,6 +16,16 @@ $secondaryColorDisabled: #000000;
 $grey: #5A6978;
 $lightGrey: #B6B6B6;
 
+//gradient colors
```
Comment (CodingCarlos) [inline sass/base/_variables.scss]: From here to line 24, what are this things doing here? We're not using any blue gradient, as far as I know. Also, we already have background colors (both in normal and light version) some lines down (30 and 31). 

Also are `maintag` and `mainQuestion` variable names accurate? Aren't just other way to use `textDark`?

File: languages/en_us.js [hunk 2]
```diff
@@ -42,8 +42,13 @@
...
 			ERROR_TERMS_EMPTY: 'You must accept the terms and conditions of the service',
 			ERROR_RECAPTCHA_NOT_CHECKED: 'This app only is able to be used by humans. Aren\'t you human?',
+			GALLERY_FEATURED_VIDEOS: 'Featured videos',
+			GALLERY_VIEW_ALL: 'View all',
+			GALLERY_ORDER_BY: 'Order by',
+			GALLERY_ORDER_BY_SELECTOR_RELEVANCE: 'Relevance',
+			VIDEO_CARD_VERIFIED: 'Verified',
+			VIDEO_CARD_NOT_VERIFIED: 'Not verified',
 			DOWNLOAD_CODE: 'Enter your download code',
-			VERIFIED: 'Verified video',
```
Comment (CodingCarlos) [inline languages/en_us.js]: What happens with [video-download view (line 26)](https://github.com/Videona/platform-frontend/blob/dev/pages/video-download/video-download.view.html#L26) when you remove this line?

File: languages/es_es.js [hunk 3]
```diff
@@ -42,8 +42,13 @@
...
 			ERROR_TERMS_EMPTY: 'Tienes que aceptar los términos y condiciones de uso.',
 			ERROR_RECAPTCHA_NOT_CHECKED: 'En esta aplicación solo aceptamos humanos ¿Acaso eres un robot?',
+			GALLERY_FEATURED_VIDEOS: 'Featured videos',
+			GALLERY_VIEW_ALL: 'Ver todo',
+			GALLERY_ORDER_BY: 'Ordenar por',
+			GALLERY_ORDER_BY_SELECTOR_RELEVANCE: 'Relevancia',
+			VIDEO_CARD_VERIFIED: 'Verificado',
+			VIDEO_CARD_NOT_VERIFIED: 'No verificado',
 			DOWNLOAD_CODE: 'Introduce un código de descarga',
-			VERIFIED: 'Video verificado',
```
Comment (CodingCarlos) [inline languages/es_es.js]: Same question here.

**Review by CodingCarlos — CHANGES_REQUESTED** [review]

File: languages/en_us.js (hunk 2 above)
Comment (DevStarlight) [reply languages/en_us.js]: As I've seen, that deleted line is correct, but what is not correct and you're right is the calling to the correct variable in the view. At this point, we've changed VERIFIED to VIDEO_CARD_VERIFIED, so I'm updating the view with this name.

File: sass/base/_variables.scss (hunk 1 above)
Comment (DevStarlight) [reply sass/base/_variables.scss]: I've just fixed your requested changes =)

File: languages/en_us.js (hunk 2 above)
Comment (CodingCarlos) [reply languages/en_us.js]: So, if there's not only being calling in `video-card` component... Shall the name be `VIDEO_CARD_VERIFIED` or better something just like `VIDEO_VERIFIED`?

**Review by CodingCarlos — APPROVED** [review]

File: components/video-card/video-card.scss [hunk 4]
```diff
@@ -1,8 +1,9 @@
 video-card {
-	div {
+	div.card-container {
```
Comment (DevStarlight) [inline components/video-card/video-card.scss]: so... if you have defined a class... for what do you specify the `div` tag?

**Comment by DevStarlight** [comment]
<3 nice job guys! DO NOT close this PR until it's accepted

File: RELEASE [hunk 5]
```diff
@@ -1,2 +1,2 @@
-v0.0.4
-- Adding user feedback about code validity on downloading original video
+v0.1.0
```
Comment (javierLiarte) [inline RELEASE:1]: package.json version is inconsistent with 0.1.0

  "version": "0.0.4",

**Review by javierLiarte — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vimojo-platform-frontend",
-  "version": "0.0.4",
+  "version": "0.1.0",
```
Comment (javierLiarte) [inline package.json:3]: :+1:

File: RELEASE (hunk 5 above)
Comment (DevStarlight) [reply RELEASE:1]: That was fixed in 97f9750

**Review by javierLiarte — APPROVED** [review]
Great work, guys, let's move forward! :+1:

**Pull request merged**